### PR TITLE
Add removable filter in ToC top vertical bar

### PR DESCRIPTION
**Pull request by samshara**

### CODE REVIEW

File: src/views/Entries/QualityControl/Chip/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <Icon
+                name={icon}
+                className={iconClassName}
+            />
+            }
+            {isTruthyString(label) &&
+            <div className={_cs(styles.label, labelClassName)}>
+                {label}
+            </div>
+            }
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/Chip/index.tsx]: Fix indentation

File: src/views/Entries/QualityControl/index.tsx [hunk 2]
```diff
@@ -319,10 +319,17 @@ function QualityControl(props: Props) {
         handleEntryDelete,
     ]);
 
-    const tocFilterRendererParams = useCallback((_: string, data: MatrixKeyId) => ({
-        className: styles.badge,
-        title: labelSelector(data),
-    }), []);
+    const handleRemoveSelection = useCallback((id: string) => {
+        const newTocFilters = tocFilters.filter(f => idSelector(f) !== id);
+        setTocFilters({ tocFilters: newTocFilters });
```
Comment (AdityaKhatri) [inline src/views/Entries/QualityControl/index.tsx:324]:
```suggestion
        setTocFilters((oldToCFilters) => oldToCFilters.filter(f => idSelector(f) !== id));
```

Comment (samshara) [reply src/views/Entries/QualityControl/index.tsx:324]: here setTocFilters is a redux function, not a useState update function.

Comment (AdityaKhatri) [reply src/views/Entries/QualityControl/index.tsx:324]: Sounds good!

**Pull request merged**